qa/workunits/rbd/cli_generic.sh

```diff
@@ -1264,6 +1264,14 @@ test_mirror_snapshot_schedule() {
     rbd mirror snapshot schedule add 1h 00:15
     test "$(rbd mirror snapshot schedule ls)" = 'every 1h starting at 00:15:00'
 
+    # Negative tests
+    expect_fail rbd mirror snapshot schedule add dummy
+    expect_fail rbd mirror snapshot schedule add -p rbd2/ns1 --image test1 dummy
+    expect_fail rbd mirror snapshot schedule remove dummy
+    expect_fail rbd mirror snapshot schedule remove -p rbd2/ns1 --image test1 dummy
+    test "$(rbd mirror snapshot schedule ls --image rbd2/ns1/test1)" != 'every 5m'
```

Comment:
what is the purpose of this test?

Reply:
Just few negative tests validation for scheduler.

Reply:
Stil I can't understand this `!= 'every 5m'` test. Why it is `5m`? Is there a way it could (wrongly) be `every 5m` here?

Reply:
Just making sure schedule does not exists before trying to remove in the next line. (1273)